### PR TITLE
Fix codecs

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

File: alembic/versions/ea74eca400ce_sip_to_pjsip_endpoint.py [hunk 1]
```diff
@@ -694,7 +696,31 @@ def _add_option_no_duplicate(self, kv):
...
+                self._codecs.append(codec)
+        return
+        yield
+
+    def _convert_deny(self, val):
+        for codec in val.split(','):
+            if codec == 'all':
+                self._codecs = ['!all']
+            elif codec in self._codecs:
+                self._codecs.remove(codec)
```
Comment (fblackburn1) [inline alembic/versions/ea74eca400ce_sip_to_pjsip_endpoint.py:721]: I didn't test it but if you have
```
allow=ulaw,ulaw
deny=ulaw
```
the result will be
```
codec=ulaw
```
which is not the behavior that we want (but maybe too corner case)?

File: alembic/versions/ea74eca400ce_sip_to_pjsip_endpoint.py [hunk 2]
```diff
@@ -694,7 +696,31 @@ def _add_option_no_duplicate(self, kv):
...
+        return
+        yield
```
Comment (fblackburn1) [inline alembic/versions/ea74eca400ce_sip_to_pjsip_endpoint.py:723]: Is it valid?

File: alembic/versions/ea74eca400ce_sip_to_pjsip_endpoint.py (hunk 1 above)
Comment (pc-m) [reply alembic/versions/ea74eca400ce_sip_to_pjsip_endpoint.py]: Fixed, I added a loop to remove all occurrences of codec

File: alembic/versions/ea74eca400ce_sip_to_pjsip_endpoint.py (hunk 2 above)
Comment (pc-m) [reply alembic/versions/ea74eca400ce_sip_to_pjsip_endpoint.py:724]: yes, it turns the function into a generator which is required for the loop and since it returns before the yield no value will ever be generated

Comment (fblackburn1) [reply alembic/versions/ea74eca400ce_sip_to_pjsip_endpoint.py:724]: Oh I didn't know, thank you
